src/cmd/root.go

```diff
@@ -98,12 +99,20 @@ func initConfig() {
 	}
 }
 
+func checkFileExists(filePath string) bool {
+	_, err := os.Stat(filePath)
+	return !errors.Is(err, os.ErrNotExist)
+}
+
 func readConfig() error {
 	if cfgFile != "" {
 		if cfgFile == "." {
 			viper.SetConfigType("yaml")
 			return viper.ReadConfig(os.Stdin)
 		} else {
+			if !checkFileExists(cfgFile) {
```

Comment:
I don't love this but currently if you run `go run main.go run --mode faktory` it crashes because there is no `./opslevel.yml` file for the viper config to read.

Maybe i need to go dig and see how the CLI does this but however we changed this we completely broke the "startup" and this is the only way I could think to easily fix it.